src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -190,12 +189,18 @@
                 : (Debugger.IsAttached) ? 1 : Environment.ProcessorCount;
 
             var channelOptions = new BoundedChannelOptions(2000)
+            {
+                SingleWriter = true,
+                SingleReader = true,
+            };
+            readyToHashChannel = Channel.CreateBounded<int>(channelOptions);
```

Comment:
>readyToHashChannel [](http://example.com/codeflow?start=12&length=18)

Currently the hashing channel is single-threaded. There is an opportunity here, I think, the improve performance by making this phase multi-threaded.